Objective-C/CBLCollection.mm

```diff
@@ -951,4 +952,30 @@ - (FLSliceResult) emptyFLSliceResult: (CBLDatabase*)db {
     return result;
 }
 
+- (nullable CBLQueryIndex*) getIndex:(nonnull NSString*)name error: (NSError**)error {
+    CBLAssertNotNil(name);
+    
+    CBL_LOCK(_mutex) {
+        if (![self checkIsValid: error])
+            return nil;
+        
+        C4Error* c4err = {};
+        CBLStringBytes iName(name);
```

Comment:
(What is `i` stand for?) maybe `sName`?

Reply:
index name - I've checked around and seems like `sName` is scope name and `cName` is collectionName. Also `iName` is used in other places such as `createIndex`